### PR TITLE
feat: add pl_PL localization

### PR DESCRIPTION
**Pull request by mavethee**
Initial polish localization for shadPS4 :3

### CODE REVIEW

File: src/qt_gui/translations/pl_PL.ts [hunk 1]
```diff
@@ -1,491 +1,493 @@
...
+    </message>
+    <message>
+        <location filename="../gui_context_menus.h" line="41"/>
+        <source>SFO Viewer</source>
+        <translation>Menedżer plików SFO</translation>
+    </message>
+    <message>
+        <location filename="../gui_context_menus.h" line="42"/>
+        <source>Trophy Viewer</source>
+        <translation>Menedżer trofeuów</translation>
```
Comment (wojek07) [inline src/qt_gui/translations/pl_PL.ts]:
```suggestion
        <translation>Menedżer trofeów</translation>
```
"trofeów" is genitive, plural form for this noun, right? Some ref in polish: https://wsjp.pl/haslo/podglad/26899/trofeum/2584240/sportowe
Nit: If this is just a preview/browser of collected/avaliable trophies, perhaps "Przeglądarka trofeów" may be more appropiate, even thought its direct english equivalent will be "trophy browser".

File: src/qt_gui/translations/pl_PL.ts [hunk 2]
```diff
@@ -1,491 +1,493 @@
...
+    </message>
+    <message>
+        <location filename="../gui_context_menus.h" line="173"/>
+        <source>Shortcut creation</source>
+        <translation>Tworzenie skrótu</translation>
+    </message>
+    <message>
+        <location filename="../gui_context_menus.h" line="174"/>
+        <source>Shortcut created successfully!\n %1</source>
+        <translation>Utworzenie skrótu zakończone pomyślnie!\n %1</translation>
```
Comment (wojek07) [inline src/qt_gui/translations/pl_PL.ts:133]: Nit: "Skrót utworzono pomyślnie!\n %1"

File: src/qt_gui/translations/pl_PL.ts [hunk 3]
```diff
@@ -1,491 +1,493 @@
...
+        <source>Open Folder</source>
+        <translation>Open Folder</translation>
+    </message>
+</context>
+<context>
+    <name>TrophyViewer</name>
+    <message>
+        <location filename="../trophy_viewer.cpp" line="8"/>
+        <source>Trophy Viewer</source>
+        <translation>Menedżer trofeuów</translation>
```
Comment (wojek07) [inline src/qt_gui/translations/pl_PL.ts]:
```suggestion
        <translation>Menedżer trofeów</translation>
```

File: src/qt_gui/translations/pl_PL.ts [hunk 4]
```diff
@@ -1,491 +1,493 @@
...
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="453"/>
+        <source>Advanced</source>
+        <translation>Zaawansowane</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="462"/>
+        <source>Enable Shaders Dumping</source>
+        <translation>Włącz zgrywanie tekstur</translation>
```
Comment (wojek07) [inline src/qt_gui/translations/pl_PL.ts]:
```suggestion
        <translation>Włącz zgrywanie shaderów</translation>
```
There is small not-insignificant difference between shaders and textures.

File: src/qt_gui/translations/pl_PL.ts (hunk 1 above)
Comment (mavethee) [reply src/qt_gui/translations/pl_PL.ts]: I've picked `explorer` for for all, as it's more fitting.

File: src/qt_gui/translations/pl_PL.ts (hunk 3 above)
Comment (mavethee) [reply src/qt_gui/translations/pl_PL.ts]: Yeah, typo, mb.

File: src/qt_gui/translations/pl_PL.ts (hunk 4 above)
Comment (mavethee) [reply src/qt_gui/translations/pl_PL.ts]: `shaders` are `cienie`, don't suggest random ass polished english words, please.
Not sure what I wrote `textures` o.0

File: src/qt_gui/translations/pl_PL.ts (hunk 2 above)
Comment (mavethee) [reply src/qt_gui/translations/pl_PL.ts:133]: Bezokoliczniki >>>

**Comment by georgemoralis** [comment]
since there is also #582  , you two should work together , i can only merge one and i don't know polish so i can verify it :D

**Comment by mavethee** [comment]
> since there is also #582 , you two should work together , i can only merge one and i don't know polish so i can verify it :D

Define work together :0
It's strings for half an hour of work.
Used Linguist so ts file is more proper than basics dima did.
In future, slight suggestion to use `type=unfinished` instead of filling translated values to avoid confusion, example below from my DuckStation's file.

```ts
    <name>Achievements</name>
    <message>
        <location filename="../../core/system.cpp" line="1680"/>
        <location filename="../../core/system.cpp" line="1694"/>
        <source>Overriding executable</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
```

**Comment by georgemoralis** [comment]
i must somehow decide which of 2 to merge , since there are 2 pr's with polish...

**Comment by mavethee** [comment]
> i must somehow decide which of 2 to merge , since there are 2 pr's with polish...

Ask testers I guess 🤷🏻‍♂️ 
Works for me besides small things missing, like Game List/Game Grid label when selected, OK, Cancel, Restore Defaults stuff, rest seems to be displayed properly:
CleanShot 2024-08-26 at 10 15 40@2x

**Comment by Graubek** [comment]
> i must somehow decide which of 2 to merge , since there are 2 pr's with polish...

In my opinion, it should be merged

**Pull request merged**